src/controller/InvokeInteraction.h

```diff
@@ -90,6 +90,69 @@ InvokeCommandRequest(Messaging::ExchangeManager * aExchangeMgr, const SessionHan
     return CHIP_NO_ERROR;
 }
 
+/*
+ * A typed group command invocation function that takes as input a cluster-object representation of a command request and
+ * callbacks when completed trought the done callback
+ *
+ * The RequestObjectT is generally expected to be a ClusterName::Commands::CommandName::Type struct, but any object
+ * that can be encoded using the DataModel::Encode machinery and exposes the GetClusterId() and GetCommandId() functions
+ * and a ResponseType type is expected to work.
+ *
+ * Since this sends a group command, no response will be received and all allocated rescources will be cleared before exing this
+ * function
+ */
+template <typename RequestObjectT>
+CHIP_ERROR
+InvokeGroupCommandRequest(Messaging::ExchangeManager * exchangeMgr, chip::FabricIndex fabric, chip::GroupId groupId,
+                          chip::NodeId sourceNodeId, const RequestObjectT & requestCommandData,
+                          typename TypedCommandCallback<typename RequestObjectT::ResponseType>::OnSuccessCallbackType onSuccessCb,
+                          typename TypedCommandCallback<typename RequestObjectT::ResponseType>::OnErrorCallbackType onErrorCb,
+                          typename TypedCommandCallback<typename RequestObjectT::ResponseType>::OnDoneCallbackType onDoneCb)
+{
+    app::CommandPathParams commandPath = { 0 /* endpoint */, groupId, RequestObjectT::GetClusterId(),
+                                           RequestObjectT::GetCommandId(), (app::CommandPathFlags::kGroupIdValid) };
```

Comment:
are the brackets needed around kGroupIdValid?

Reply:
done